src/node.rs

```diff
@@ -320,3 +327,14 @@ impl Allocator {
         Id(self.last_id.fetch_add(1, Ordering::Relaxed))
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn measure_func_is_send_and_sync() {
+        fn is_send_and_sync<T: Sync>() {}
+        is_send_and_sync::<MeasureFunc>();
```

Comment:
This test is missing the `T: Send` bound.

Reply:
Good eye. I'll fix this ASAP.